google-cloud-spanner-hibernate-testing/src/test/java/org/hibernate/userguide/sql/SQLTest.java

```diff
@@ -541,7 +502,7 @@ public void test_sql_hibernate_dto_query_example() {
 			Session session = entityManager.unwrap( Session.class );
 			//tag::sql-hibernate-dto-query-example[]
 			List<PersonSummaryDTO> dtos = session.createNativeQuery(
-				"SELECT p.id as \"id\", p.name as \"name\" " +
+				"SELECT p.id as id, p.name as name " +
```

Comment:
Again, something we don't support but every other RDBMS does?

Reply:
Yes this is the same quoted aliases that spanner doesn't support (but others do to support aliases with spaces, see #47)